docs/standard/commandline/design-guidance.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Command-line design guidance for System.CommandLine
+description: "Provides guidance for designing a command-line interface."
+ms.date: 06/16/2025
+no-loc: [System.CommandLine]
+helpviewer_keywords:
+  - "command line"
+  - "System.CommandLine"
+ms.topic: conceptual
+---
+
+# Design guidance
```

Comment:
I've just moved this text, the source: https://github.com/dotnet/docs/blob/main/docs/standard/commandline/syntax.md#design-guidance